api/spec/v1.yaml

```diff
@@ -1933,6 +1970,61 @@ components:
         self_delegation_shares:
           allOf: [$ref: '#/components/schemas/TextBigInt']
           description: The shares of tokens this validator has delegated to itself, and are NOT in the process of debonding.
+        active_balance_24:
```

Comment:
Note: This attempts to find the balance 24 epochs ago, which isn't exactly 24 hours but it's accurate enough. Also open to field name suggestions here.